comment/models/reactions.py

```diff
@@ -0,0 +1,199 @@
+from enum import IntEnum, unique
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.db.models.signals import post_delete
+from django.dispatch import receiver
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from comment.manager.reactions import ReactionManager
+from comment.models import Comment
+
+
+class Reaction(models.Model):
+
+    comment = models.ForeignKey(Comment, related_name='reactions', on_delete=models.CASCADE)
+    likes = models.PositiveIntegerField(default=0)
+    dislikes = models.PositiveIntegerField(default=0)
+    
+    objects = models.Manager()
+    comment_objects = ReactionManager()
+    
+    def _increase_likes(self):
```

Comment:
In my openion all this model's methods are not private since all of them will be called outside the class.

Reply:
Only `increase_reaction_count` and `decrease_reaction_count` were called from outside the class. I have made them public now.